src/data_readers/data_reader_jag_conduit.cpp

```diff
@@ -1248,13 +1285,17 @@ std::vector<data_reader_jag_conduit::input_t> data_reader_jag_conduit::get_input
                       + " in conduit node: " + std::to_string(sample_id));
         }
         conduit::Node n_input;
-        load_conduit_node(sample_id, conduit_field, n_input);
-        sample[conduit_obj].set(n_input);
+        bool from_file = load_conduit_node(sample_id, conduit_field, n_input);
+        if (from_file) {
+          sample[conduit_obj].set(n_input);
+        } else {
+          sample = n_input;
+        }
+        const input_t val_raw = static_cast<input_t>(sample[conduit_obj].value());
+        const input_t val = static_cast<input_t>(val_raw * tr->first + tr->second);
+        inputs.push_back(val);
+        tr ++;
```

Comment:
This has been put back in at the wrong level of the if-block.  It needs to be outside of the "if(sample[conduit_obj].schema().dtype().is_empty()) {" conditional so that it will be used when grabbing samples from the data store as well.